cpmpy/solvers/choco.py

```diff
@@ -175,6 +175,9 @@ def solve(self, time_limit=None, **kwargs):
             # translate objective
             if self.has_objective():
                 self.objective_value_ = sol.get_int_val(self.solver_var(self.obj))
+        else: # clear values of variables
+            for cpm_var in self.user_vars:
+                cpm_var.clear()
```

Comment:
I guess we also have to put None to objective value??

Also, it is possible that vars have already value, and then the user changes something and uses solveAll(). So, in that case we also need to empty them? As I see in the solveAll() code, it normally has the value of the last solution found...